crates/db_schema/src/schema.rs

```diff
@@ -805,19 +787,15 @@ diesel::table! {
 diesel::table! {
     site (id) {
         id -> Int4,
-        #[max_length = 20]
         name -> Varchar,
         sidebar -> Nullable<Text>,
         published -> Timestamptz,
         updated -> Nullable<Timestamptz>,
         icon -> Nullable<Text>,
         banner -> Nullable<Text>,
-        #[max_length = 150]
         description -> Nullable<Varchar>,
-        #[max_length = 255]
         actor_id -> Varchar,
         last_refreshed_at -> Timestamptz,
-        #[max_length = 255]
```

Comment:
Something went wrong here, these annotations should stay.

Reply:
Weird. this file was autogenerated by my diesel migration. do you guys manually maintain this file?

Reply:
No it works with autogeneration. Maybe you are using an older version of diesel-cli?

Reply:
Yeah that was it. fixed